chips/atmega2560-hal/Cargo.toml

```
@@ -0,0 +1,15 @@
+[package]
+name = "atmega2560-hal"
+version = "0.1.0"
+authors = ["Andrew Dona-Couch <avr-hal@couchand.com>"]
+edition = "2018"
+
+[features]
+rt = ["avr-device/rt"]
+
+[dependencies]
+avr-hal-generic = { path = "../../avr-hal-generic/" }
+
+[dependencies.avr-device]
+version = "0.1.0"
```

Update was released!

```suggestion
version = "0.1.1"
```

---

Done.